src/components/UI/Overlay/OverlayContent/index.tsx

```diff
@@ -21,9 +21,11 @@ export const OverlayContent = ({ className, title, titleIcon, children, ...props
       <div className={`overlay-content ${className}`} {...props}>
         <div className="overlay-header">
           <div className="row no-gutters align-items-center">
-            {titleIcon && (
+            {isTitleIconSuccess !== undefined && (
```

Comment:
```suggestion
            {isSuccess && (
```

Reply:
isSuccess not defined = will not render any icon beside the title, then if isSuccess is defined, it will determine whether the x or checked icon will show

![Screenshot 2020-05-05 at 4 27 34 PM](https://user-images.githubusercontent.com/4774314/81047632-68828900-8eed-11ea-8517-5f13fd27ba4b.png)
![Screenshot 2020-05-05 at 4 27 38 PM](https://user-images.githubusercontent.com/4774314/81047636-6ae4e300-8eed-11ea-8f34-60d07c91c758.png)